myst_nbs/case_studies/stochastic_volatility.myst.md

```diff
@@ -6,13 +6,20 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3 (ipykernel)
+  display_name: pymc_env
   language: python
-  name: python3
+  name: pymc_env
 ---
 
+(notebook_name)=
 # Stochastic Volatility model
 
+:::{post} June 17, 2022
+:tags: time series
+:category: basic
+:author: Thomas Wiecki
```

Comment:
Everyone who appears on the authorship section (except those that have only re-executed the notebook) should appear here.

Reply:
This brings up an interesting issue because there is extensive history on this notebook stretching back to its `.py` incarnation, with multiple people making multiple edits over time. I've got a new author list at the `Authors` section at the end of this notebook - does that format look acceptable? 

For more recent edits, I've tried to explicitly point out the PR while for those that edited multiple times, I was less specific.